### PR TITLE
Fix legacyskip

### PR DESCRIPTION
**Pull request by AutumnClove**
After creating your pull request, tick these boxes if they are applicable to you.

- [x] I have tested my changes against the `review` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [x] @Roadhog360  and I have tested my changes on Python 3.5.3 or higher

----

### Description
Fix for legacyskip changes suggested by @TheerapakG
Issue bought up by @Roadhog360

### CODE REVIEW

**Comment by TheerapakG** [comment]
That doesn't seem right. Why check if both forced and legacy?

**Comment by AutumnClove** [comment]
@TheerapakG doing and not doesn't let users force skip still and vote skip, doing or just raises perms error, doing this allows users to do !skip f with legacy skip on and force skip or do

File: musicbot/bot.py [hunk 1]
```diff
@@ -2088,7 +2088,7 @@ def check(reaction, user):
                 player.skip()  # TODO: check autopause stuff here
                 await self._manual_delete_check(message)
                 return Response(self.str.get('cmd-skip-force', 'Force skipped `{}`.').format(current_entry.title), reply=True, delete_after=30)
-            else:
+            if(param.lower() in ['force', 'f']):
```
Comment (jayktaylor) [inline musicbot/bot.py]: This logic is messy and should be rewritten rather than repeating code.

**Review by jayktaylor — CHANGES_REQUESTED** [review]

File: musicbot/bot.py (hunk 1 above)
Comment (TheerapakG) [reply musicbot/bot.py]: Does the rewrite acceptable?

**Review by jayktaylor — APPROVED** [review]
That's cleaner, thanks

**Pull request merged**